doc/1.Version.md

```diff
@@ -54,7 +54,7 @@ Assertion 1.2.3:
     SpdmMessage.SPDMVersion == 0x10
 
 Assertion 1.2.4:
-    SpdmMessage.Param1 == InvalidRequest.
+    (SpdmMessage.Param1 == InvalidRequest) OR (SpdmMessage.Param1 == VERSION_MISMATCH)

```

Comment:
We need use `VersionMismatch` in the spec.

Reply:
Thanks. I have changed it.